middle_end/flambda/compilenv_deps/coercion.ml

```diff
@@ -0,0 +1,74 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                   Mark Shinwell, Jane Street Europe                    *)
+(*                                                                        *)
+(*   Copyright 2019 Jane Street Group LLC                                 *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
```

Comment:
Let's add `open! Int_replace_polymorphic_compare`.  Also please copy the warning stanza from the .mli file (having removed number 4 from the exclusion list) and fix any resulting warnings.

Reply:
Done (there was indeed a warning).